Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -1,7 +1,104 @@
 ﻿using System;
-
+using System.Text.RegularExpressions;
 namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
 {
+    class Equation{
+        public Equation(string operand1,string operand2,string operation,string result){
+            this.operand1=operand1;
+            this.operand2=operand2;
+            this.operation=operation;
+            this.result=result;
+        }
+        public enum EquationParts{
+            operand1,operand2,result
+        }
+        private string operand1;
+        private string operand2;
+        private string operation;
+        private string result;
+        public string Operand1 { get; private set; }
+        public string Operand2 { get; private set; }
+        public string Operation { get; private set; }
+        public string Result { get; private set; }
+        private bool IsValidValue(string value,string expectedValue){
+            if(expectedValue.Length!=value.Length)
+            return false;
+            return true;
+        }
+        private EquationParts GetEquationPartToBeCorrected(){
+            if(operand1.Contains('?'))
+            return EquationParts.operand1;
+            else if(operand2.Contains('?'))
+            return EquationParts.operand2;
+            else
+            return EquationParts.result;

```

Comment:
IF result none of Operand1/Operand2/Result contains ? then???
You should have one more value in EquationParts, "Invalid" and if none of parts contains ? then Invalid should be returned